docs/source/gbd2017_models/causes/neoplasms/breast_cancer_with_stage0/cancer_model.rst

```diff
@@ -0,0 +1,319 @@
+(such as being only fatal or only nonfatal), as well as restrictions on the age
+and sex of simulants to which different aspects of the cause model apply.
+
+.. list-table:: Restrictions
+   :widths: 15 15 20
+   :header-rows: 1
+
+   * - Restriction Type
+     - Value
+     - Notes
+   * - Male only
+     - False
+     -
+   * - Female only
+     - False
+     -
+   * - YLL only
+     - False
+     -
+   * - YLD only
+     - False
+     -
+   * - YLL age group start
+     - 15 to 19
+     - GBD age group id 8
+   * - YLL age group end
+     - 95 plus
+     - GBD age group id 235
+   * - YLD age group start
+     - 15 to 19
+     - GBD age group id 8
+   * - YLD age group end
+     - 95 plus
+     - GBD age group id 235
+
+
+Vivarium Modeling Strategy
+--------------------------
+
+Things to consider: 
+
+1. Within GBD 2017 data, there is no remission rate for stage 1+ breast cancer.
+2. Within GBD 2017, after diagnosis/treatment if a patient survives more than 10 years, they are considered cured for calculating disability. Additionally, per GBD 2017, the patients also do not have excess mortality rate after 10 years. For vivarium simulation models, this means we need to track how long patients have been in the breast cancer state. This should be easy for those who has a breast cancer incidence after sim start. For those who were iniatialized in the breast cancer state, we need to assign when breast cancer was diagnosised in the past. After their 10 years is up and they still havent died, we need to decide what susceptability state to remit them to. 
+
+.. todo::
+
+   Add more assumptions and limitations.
+
+
+Compartmental Diagram
++++++++++++++++++++++
+
+  .. image:: compartmental_model_simple.svg
+
+
+State and Transition Data Tables
+++++++++++++++++++++++++++++++++
+
++----------------------------------------------------------------------------+
+| State definitions                                                          |
++=======================+================+===================================+ 
+| State                 | State name     | Definition                        |
++-----------------------+----------------+-----------------------------------+
+| S                     | Susceptible    | Susceptible to DCIS or LCIS       |
++-----------------------+----------------+-----------------------------------+
+| DCIS                  | with condition | with condition DCIS               |
++-----------------------+----------------+-----------------------------------+
+| LCIS                  | with condition | with condition LCIS               |
++-----------------------+----------------+-----------------------------------+
+| BC                    | with condition | with condition breast cancer      |
++-----------------------+----------------+-----------------------------------+
+
+.. list-table:: State Data
+   :widths: 10 15 15 35 25
+   :header-rows: 1
+   
+   * - State
+     - Measure
+     - Value
+     - Sources
```

Comment:
In other cause models, we have a separate table called Data Sources that defines all the variables in the State Data and Transition Data tables. The general philosophy for that design is to keep everything in "tidy data" format: https://vita.had.co.nz/papers/tidy-data.pdf

In particular, having a separate table will make it easier to specify which data source is for which variable/parameter when there are multiple variables used in a formula, and it collects all the parameters in one place so that it is easy to see all of the data sources you will need in one place.

Reply:
Oh, it looks like Rajan already mentioned this.

Reply:
just separated them :) thanks